runtime/doc/news.txt

```diff
@@ -54,7 +54,9 @@ EVENTS
 
 LSP
 
-• TODO
+• The watchfunc backend was implemented with fswatch, but deactivated by
```

Comment:
Not sure if we need to mention this here since it was deactivated by default, and so it's not like a breaking change or anything.

Reply:
Where do I activate it by default? :-)

Reply:
Found it and enabled it.

Reply:
I don't think we want to enable it. Anything that uses inotify is going to have the same problems. It wasn't startup-time of fswatch that was the issue.

Reply:
What was the problem?